react/README.md

```diff
@@ -386,7 +386,7 @@
 
     // good
     <Foo
-      ref={(ref) => { this.myRef = ref; }}
+      ref={ref => this.myRef = ref}
```

Comment:
this is invalid; implicit return arrow functions should not contain assignments; so this must be an explicit return; one-arg explicit return arrow functions must have parens around the arg. In other words, it's perfect as-is.